src/napari_plugin_manager/qt_warning_dialog.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, parent: QWidget) -> None:
         self.setWindowTitle('Restart napari')
         okay_btn = QPushButton('Okay')
         self.restart_warning_text = """
-Plugins have been installed or uninstalled. If you notice any
+Plugins have been installed, uninstalled or updated. If you notice any
 issues with plugin functionality, you may need to restart napari.
         """
```

Comment:
Two suggestions:

- Maybe we can use `textwrap.dedent()` to keep the code indentation the same.
- Can we just say "Plugins have been modified"?

Reply:
> Can we just say "Plugins have been modified"?

"modified" sounds weird to me. I prefer the current state. Can make it shorter with:
`added/removed or updated`
but not sure how necessary that is.

Reply:
`Plugins have been modified` or `Plugins have been added/removed or updated` sounds good to me (even the initial one, although rather long, sounded good to me 😅) so up to you folks :)